benchmark/Streamly/Benchmark/Prelude/Serial.hs

```diff
@@ -512,8 +593,11 @@ o_1_space_elimination_folds value =
         , benchIOSink value "minimumBy" minimumBy
         , benchIOSink value "minimum" minimum
 
+        , bench "the" $ nfIO $ randomRIO (1,1) >>= the . repeat value
         , benchIOSink value "find" (find value)
-    -- , benchIOSink value "lookup" lookup
+        , benchIOSink value "lookupFirst" (lookup 1)
```

Comment:
This is trivial, the timing will be tiny, pretty much useless.

Reply:
and it will also make each benchmark reported in nanoseconds, very large numbers.